tools/protodoc/templates/extension.rst.tpl

```diff
@@ -13,3 +13,15 @@ This extension may be referenced by the qualified name ``{{extension}}``
 {% for cat in categories %}
   - :ref:`{{cat}} <extension_category_{{cat}}>`
 {% endfor %}
+
+{% if type_urls|length > 0 %}
+  This extension can be configured with the following type URLs:
```

Comment:
my main concern here is that its not clear what this means - either if you are coming from a position of zero-knowledge, or trying to discriminate/associate/understand `type_urls` and extension `name`

in most cases at least, would it not be more correct to say that this extensions _must_ be configured with this type URL ?

Reply:
Good point, changing to _must_.